maven_iuvs/graphics/graphics.py

```diff
@@ -1304,3 +1327,102 @@ def reset_symlog_labels(fig, axes):
         # set tick padding above the label
         for tick in ax.get_xaxis().get_major_ticks():
             tick.set_pad(11)
+
+
+def make_sza_plot(ax, fitfile, linecolor="cornflowerblue"):
```

Comment:
would be nice if ax was an optional argument so these could be called directly on a fits file, but that's not needed now.

Reply:
Easy enough, fixed by 48e969aa8240f4d5c25b2e7a76aeeb796a9e5b50